tests/wamr-test-suites/spec-test-script/runtest.py

```diff
@@ -101,35 +135,53 @@ def __init__(self, args, no_pty=False):
             self.stdin = os.fdopen(master, 'r+b', 0)
             self.stdout = self.stdin
 
+        if platform.system().lower() == "windows":
```

Comment:
Just be curious, is it possible to have a cross-platform solution for reading stdout?

Reply:
Good question. The only cross-platform I could think of was by using a thread and a queue (just like it was implemented in this PR for Windows); but I did not use this technique for other operating systems because it turned to be significantly (2 - 3 x) slower, so I kept the approach with `select` for non-windows platforms.